lib/Mojolicious/Plugin/AssetPack/Pipe/OptiPng.pm

```diff
@@ -0,0 +1,94 @@
+
+package Mojolicious::Plugin::AssetPack::Pipe::OptiPng;
```

Comment:
I think you can just call it "Png" or maybe "ImagePng"... The other pipes are called "Css", "JavaScript", ... which reflects on the files it can handle, not what it does.